### DISCUSSION TITLE
Geldtransit Sphäre

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#526] [General]
Ich richte mir gerade die Buchungsarten für SKR42 ein und buche danach. Bei "1372 0 Geldtransit" habe ich bisher keine Buchungsklasse ausgewählt. Das funktioniert bei der Erstellung der Buchungsart, da "Keine feste Zuordnung von Buchungsklasse zu Buchungsart z. B. SKR 42" aktiviert ist.
Wenn ich die Buchungsart jedoch zuordnen möchte, funktioniert das nur, wenn ich auch eine Buchungsklasse auswähle. Im ersten Anlauf hätte ich jetzt "Sammelposten" gewählt, da "noch zu klärende oder die Gesamtkörperschaft betreffenden Positionen".
Würdet ihr das auch so machen?

Und wie schaut es mit durchlaufenden Posten aus? Diese sind eigentlich auch Sammelposten, richtig?

**Comment by JohannMaierhofer** [comment]
Das habe ich inkonsistent gemacht. Beim Buchungsart zuordnen muss man die Buchungsklasse eingeben. Wenn man die Buchung selbst editiert muss man sie nicht setzten und kann die Buchung speichern.

Soll ich das ändern und im Buchungsart Zuornung Dialog auch die Buchungsklasse optional machen?

**Reply by tolot27** [reply]
> Soll ich das ändern und im Buchungsart Zuornung Dialog auch die Buchungsklasse optional machen?

Ich glaub das ist schon ganz gut und richtig so, sollte vielleicht nur dokumentiert werden. Ich kenne mich zu wenig mit Buchhaltung aus, als das ich eine generelle Aussage über eine notwendige Zuordnung einer Sphäre machen kann.
Im vorliegenden Fall ist "Sammelposten" jedoch richtig, da es "die Gesamtkörperschaft betreffend" ist.

Ich wollte nur wissen, wie ihr das macht. Welche Buchungsklasse habt ihr bei Geldtransit hinterlegt?
